packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2615,26 +2613,34 @@ export class ContainerRuntime
 		// Whether or not the message appears to be a runtime message from an up-to-date client.
 		// It may be a legacy runtime message (ie already unpacked and ContainerMessageType)
 		// or something different, like a system message.
-		const modernRuntimeMessage = messageArg.type === MessageType.Operation;
+		const hasModernRuntimeMessageEnvelop = messageArg.type === MessageType.Operation;
+		const isRuntimeMessage = messageArg.type in ContainerMessageType;
+		// Container runtime will only process ops that it understands
+		if (!(isRuntimeMessage || hasModernRuntimeMessageEnvelop)) {
+			return;
+		}
```

Comment:
I am proposing to return from the process() path if we receive an op that is not targeted for container runtime. Note this change is not final: I wanted to gather inputs from the team here to see if you think it would break anything. 
cc @markfields

Reply:
Eventually we may get here. But can't do it yet. If you are wanting to build momentum in this direction, then I would duplicate the function like we discussed in standup recently.

i.e. rather than doing nothing and returning, call a function called something like `observeNonRuntimeMessage` and copy everything from `processCore` over there (except `validateAndProcessRuntimeMessage`).  Then we can have subsequent PRs as-needed to slowly remove logic from there.

Reply:
@markfields I also realized that the running summarizer depends on the ops from runtime, so the emit op event would still be required. I don't wish to disturb the code too much at this point so will keep it simple and revert these bold changes. I got too excited to clean up I guess.